src/stirling/source_connectors/dynamic_tracer/dynamic_tracing/dwarvifier_test.cc

```diff
@@ -1807,21 +1807,21 @@ probes {
     scalar_var {
       name: "main__IntStruct_sym_addr1"
       type: UINT64
-      constant: "4989600"
+      constant: "4989632"
     }
   }
   vars {
     scalar_var {
       name: "runtime__errorString_sym_addr2"
       type: UINT64
-      constant: "4989792"
+      constant: "4989824"
     }
   }
   vars {
     scalar_var {
       name: "internal___poll__errNetClosing_sym_addr3"
       type: UINT64
-      constant: "4989824"
+      constant: "4989856"
```

Comment:
These constant changes are because changing the default go sdk caused our go 1.21 `go_cross_binary` targets to use a newer minor version:
```
(ddelnano/upgrade-go-to-1.24) $ go version bazel-bin/src/stirling/obj_tools/testdata/go/test_go_1_21_binary
bazel-bin/src/stirling/obj_tools/testdata/go/test_go_1_21_binary: go1.21.13 X:nocoverageredesign

(main) $ bazel build //src/stirling/obj_tools/testdata/go:test_go_1_21_binary
(main) $ go version bazel-bin/src/stirling/obj_tools/testdata/go/test_go_1_21_binary
bazel-bin/src/stirling/obj_tools/testdata/go/test_go_1_21_binary: go1.21.0 X:nocoverageredesign
```